apps/explorer/src/pages/object-result/views/TokenView.tsx

```diff
@@ -1,143 +1,70 @@
-						<div className="py-4 pb-7">
-							<DescriptionList>
-								{objOwner ? (
-									<DescriptionItem title="Owner" data-testid="owner">
-										{objOwner === 'Immutable' ? (
-											'Immutable'
-										) : 'Shared' in objOwner ? (
-											'Shared'
-										) : 'ObjectOwner' in objOwner ? (
-											<ObjectLink objectId={objOwner.ObjectOwner} />
-										) : (
-											<AddressLink address={objOwner.AddressOwner} />
-										)}
-									</DescriptionItem>
-								) : null}
-								<DescriptionItem title="Object ID">
-									<ObjectLink objectId={data.data?.objectId!} noTruncate />
-								</DescriptionItem>
-								<DescriptionItem title="Type">
-									{/* TODO: Support module links on `ObjectLink` */}
-									<Link to={genhref(objectType)} variant="mono">
-										{trimStdLibPrefix(objectType)}
-									</Link>
-								</DescriptionItem>
-								<DescriptionItem title="Version">
-									<Text variant="body/medium" color="steel-darker">
-										{data.data?.version}
-									</Text>
-								</DescriptionItem>
-								<DescriptionItem title="Last Transaction Block Digest">
-									<TransactionLink digest={data.data?.previousTransaction!} noTruncate />
-								</DescriptionItem>
-							</DescriptionList>
-						</div>
-						{display ? (
-							<div className="py-4 pb-7">
-								<DescriptionList>
-									<LinkOrTextDescriptionItem title="Name" value={name} />
-									<LinkOrTextDescriptionItem title="Description" value={display.description} />
-									<LinkOrTextDescriptionItem title="Creator" value={display.creator} parseUrl />
-									<LinkOrTextDescriptionItem title="Link" value={display.link} parseUrl />
-									<LinkOrTextDescriptionItem title="Website" value={display.project_url} parseUrl />
-								</DescriptionList>
-							</div>
-						) : null}
-						{storageRebate && (
-							<div className="py-4 pb-7">
-								<DescriptionList>
-									<DescriptionItem title="Storage Rebate">
-										<div className="leading-1 flex items-end gap-0.5">
-											<Text variant="body/medium" color="steel-darker">
-												{storageRebateFormatted}
-											</Text>
-											<Text variant="captionSmall/normal" color="steel">
-												{symbol}
-											</Text>
-										</div>
-									</DescriptionItem>
-								</DescriptionList>
-							</div>
-						)}
-					</div>
-					{imgUrl !== '' && (
-						<div className="min-w-0 border-0 border-t border-solid border-gray-45 pt-6 md:basis-1/3 md:border-t-0 md:pl-10">
-							<div className="flex flex-row flex-nowrap gap-5">
-								<ObjectDetails
-									image={imgUrl}
-									video={video}
-									name={name || display?.description || trimStdLibPrefix(objectType)}
-									type={video ? 'Video' : fileType ?? ''}
-									variant="large"
-								/>
-							</div>
-						</div>
+			<Tabs size="lg" value={activeTab} onValueChange={setActiveTab}>
+				<TabsList>
+					{!!fieldsCount && (
```

Comment:
Why wouldn't this just always be visible? Alternatively if you want to control this visibility here, then fetch this vs doing this fieldscount stuff

Reply:
remove to always render this